### DISCUSSION TITLE
Problems compiling for Helmet.

### DISCUSSION THREAD

**Discussion by Ranur** [hatlabs/discussions#99] [Q&A]
I managed to upload the trial software once, but when making tweaks to it, it just won't compile. I am a novice here, so any help would be greatly appreiated! I tried to revert to the the original main.cpp file, but I am still getting this error. Any suggestions?

c:/users/admin/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld.exe: cannot open map file C:/Users/Admin/VisualBåt/Projects/Sensors/HALMET-example-firmware/.pio/build/esp32dev/firmware.map: No such file or directory
collect2.exe: error: ld returned 1 exit status
*** [.pio\build\esp32dev\firmware.elf] Error 1

**Comment by big-col** [comment]
Have you tried opening the downloaded zip file again and using that set of files to upload the example firmware?

**Reply by Ranur** [reply]
Yes, and also removing the platform io files suggested in some other posts I got while googling. Now nothing works. :-( Will make a new attempt tonight to reinstall everything. I had it going so I find the errors strange.

**Reply by Ranur** [reply]
Solved! Reloaded everything including plattform io.

**Reply by big-col** [reply]
👍
